src/main/java/com/users/controller/UserController.java

```diff
@@ -27,15 +28,15 @@ public ResponseEntity<UserDto> createUser(@Valid @RequestBody UserDto user) {
     }
 
     @GetMapping()
-    public ResponseEntity<Page<UserDto>> findAll(UserCriteria criteria) {
+    public ResponseEntity<Page<UserDto>> findAll(UserCriteria criteria,@PageableDefault(size = 5) Pageable pageable) {
```

Comment:
de ce ai modificat partea cu paginarea?

Reply:
pentru ca asa era cerinta ta intitala, sa fie 5 pe pagina in cazul in care eu nu dau altceva. Daca intrebi de ce am folosit Pageable, la fel, tu mi ai zis anterior ca e mai simplu sa folosesc direct asa